libsplinter/src/network/connection_manager/messages.rs

```diff
@@ -18,8 +18,7 @@ use crate::network::connection_manager::error::ConnectionManagerError;
 
 pub enum CmMessage {
     Shutdown,
-    Subscribe(String, SyncSender<CmNotification>),
```

Comment:
Could you add some context to the commit message? I'm not sure I understand why this change is being made.

Reply:
Updated commit message. Unsubscribe is unnecessary because you'd be using notifier like an iterator and calling unsubscribe on an iterator makes not sense, since it'll stop iterating when the sender goes out of scope.